saleor/graphql/utils/__init__.py

```diff
@@ -131,3 +138,28 @@ def get_user_or_app_from_context(context):
 def requestor_is_superuser(requestor):
     """Return True if requestor is superuser."""
     return getattr(requestor, "is_superuser", False)
+
+
+def get_user_country_context(
+    destination_address: Optional[
+        Union["account_types.AddressInput", "account_models.Address"]
+    ] = None,
+    company_address: Optional["account_models.Address"] = None,
+) -> str:
+    """Get country of the current user to use for tax and stock related calculations.
+
+    User's country context is determined from the provided `destination_address` (which
+    may represent user's current location determined by the client or a shipping
+    address provided in the checkout). If `destination_address` is not given, the
+    default company address from the shop settings is assumed. If this address is not
+    set, fallback to the `DEFAULT_COUNTRY` setting.
+    """
+    if destination_address and destination_address.country:
+        if isinstance(destination_address, Country):
```

Comment:
It shouldn't be like this?
```suggestion
        if isinstance(destination_address.country, Country):
```

Reply:
Yep, it's a bug.